code/game/objects/items/devices/radio/headset.dm

```diff
@@ -251,6 +251,7 @@
 	name = "mining radio headset"
```

Comment:
Since you're making mining headsets different from cargo headsets, you might as well not make it a child type of headset_cargo and instead rename it to headset_mining